manifests/c/Contour/Terminal/0.2.0.173/Contour.Terminal.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Contour.Terminal
+PackageVersion: 0.2.0.173
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+  InstallerUrl: https://github.com/contour-terminal/contour/releases/download/v0.2.0.173/contour-0.2.0.173-win64.msi
+  InstallerSha256: 092E6952C7BEDE41D8044CBA7078D52E9F16E5B915B5FF457A6C5EF053DF0B7C
+  ProductCode: '{E92B15F2-0C08-485C-9888-A6C4A14EC8F0}'
```

Comment:
```suggestion
  ProductCode: '{88234011-14C8-4A62-8962-CCBF1B7DD62D}'
```

Reply:
I forgot where i have got the first Baker from. How did your get that? Is this changing over every version / MSI creation?

Reply:
It does seem to change for every MSI creation (now that i think about not for Firefox), i believe the developer chooses the ProductCode for exe installers (or the default might be the AppName/Moniker)